README.md

```diff
@@ -1,2 +1,72 @@
+
+In the [allstar.yaml][allstar_yaml] file we have the general configuration for allstar, like the strategy to be used, the repos to be included or excluded, and some other repo level configurations.
+
+If you need to configure allstar for your own organization you can fork this repo and modify the `yaml` files to match your needs, remember that this repo has to remain with the name `.allstar`, for more information read [Allstar installation][installation_options].
+
+## Policy configuration
+
+The application is running on all the repos of this organization with the `OptOutStrategy` strategy with the following configurations:
+
+### Branch Protection
+
+| Name of setting         | Setting | Description                                                                                                                                                                            |
+| ----------------------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Branches enforced       | Yes     | Enforce upstream branches like `main` `develop` `v*.*` → `v0.1`.                                                                                                                        |
+| Approval required       | Yes     | If required pull request reviews are enabled on the branch, you won't be able to merge changes into the branch until all requirements in the pull request review policy have been met. |
+| Approvals required      | 1       | Approval reviews required to merge changes into a branch.                                                                                                                              |
+| Block Force Push        | Yes     | You won't be able to delete or force push to the branch.                                                                                                                               |
+| Status check required   | Yes     | If required status checks are enabled on the branch, you won't be able to merge changes into the branch until all of the required CI tests pass.                                       |
+| Signed commits required | Yes     | If required commit signing is enabled on a branch, you won't be able to push any commits to the branch that are not signed and verified.                                               |
+
+`Action: issue`
+`Active repositories: public only`
+
+### Binary artifacts
+
+* No binary artifacts allowed.
+
+`Action: issue`
+`Active repositories: public only`
+
+### Outside collaborators
```

Comment:
@CristhianRodriguezMolina @SanGeeky please, split this section out for outside contributors and community members

Reply:
@CristhianRodriguezMolina ohhh yes you're right these sections match the Allstar config files. This comment does not make sense. Marked as resolved.